src/resources/postcss/tickets-admin.pcss

```
@@ -828,27 +828,25 @@ p.description {
 #ticket_order_modifier_ticket_fees {
 	width: 70%;
 
-	.automatic-fees {
-		ul {
-			display: flex;
-			flex-wrap: wrap;
-			gap: var(--tec-spacer-0);
-			list-style-type: none;
-			padding: 0;
-		}
+	.ticket_form_right {
+		max-width: 324px;
+	}
 
-		li {
-			flex: 1 0 30%;
-			margin: 0;
+	.automatic-fees, .selectable-fees {
+		margin-bottom: var(--tec-spacer-1);
+
+		strong {
+			display: inline-block;
+			margin-bottom: var(--tec-spacer-1);
 		}
-	}
 
-	.selectable-fees {
-		width: 100%;
+		.tribe-dropdown .select2-selection--multiple .select2-selection__rendered {
+			border: none;
```

We typically do `border: 0`
```suggestion
			border: 0;
```